fw/http_parser.c

```diff
@@ -3135,6 +3135,36 @@ __req_parse_x_forwarded_for(TfwHttpMsg *hm, unsigned char *data, size_t len)
 }
 STACK_FRAME_NON_STANDARD(__req_parse_x_forwarded_for);
 
+static int
+__req_parse_x_tempesta_cache(TfwHttpMsg *hm, unsigned char *data, size_t len)
+{
+	int r = CSTR_NEQ;
+	__FSM_DECLARE_VARS(hm);
+
+	__FSM_START(parser->_i_st);
+
+	__FSM_STATE(Req_I_X_Tempesta_Cache) {
+		/*
+		 * Tempesta custom header for PURGE:
+		 *	X-Tempesta-Cache = "X-Tempesta-Cache" ":" method
+		 *	method = "get"
+		 */
+		TRY_STR("get", Req_I_X_Tempesta_Cache, I_Tempesta_Cache_get);
+		__FSM_EXIT(CSTR_NEQ);
+	}
+
+	__FSM_STATE(I_Tempesta_Cache_get) {
+		if (IS_CRLF(c)) {
+			// do some processing of PURGE X-Tempesta-Cache request
```

Comment:
I believe a fair attempt was made before to avoid C++ style comments in kernel space code. As I can see now some (just a few) have still infiltrated the TLS code.